modular_skyrat/modules/mapping/code/lavaland_ruin_code.dm

```diff
@@ -81,6 +90,14 @@
 	uniform = /obj/item/clothing/under/syndicate/skyrat/tactical
 	suit = /obj/item/clothing/suit/hooded/wintercoat/syndicate
 
+/datum/outfit/lavaland_vet
```

Comment:
Blueshift: /datum/unit_test/outfit_sanity
Lost Veterinarian's backpack_contents are invalid! Couldn't add /obj/item/storage/box/survival/medical to backpack.